### DISCUSSION TITLE
/articles/microsoft-365-developer-program-status-summer-2024/

### DISCUSSION THREAD

**Discussion by giscus** [Voitanos/voitanos.io-comments#238] [Comments]
Understand the recent changes in the Microsoft 365 Developer Program, why they were made, and when you can get a free M365 developer/sandbox tenants again.
https://www.andrewconnell.com/articles/microsoft-365-developer-program-status-summer-2024/

**Comment by npc137** [comment]
Andrew, thank you so much for taking the time to write this up. This communication was the missing piece in all of this, and it is nice to have some clarity on the matter. It's a shame the program was abused enough to be locked down, but I am excited to see what a more secure option looks like ~September! Thanks again!

**Comment by dbroussa** [comment]
Thanks for the information and explanation. I've noticed that they have done something similar with Customer Immersion Experiences as well. I use those types of tenants more than Dev tenants but both have their value. I'm all for MS securing their environments and look forward to their new process.

**Reply by andrewconnell** [reply]
Yup, this happened across the board... even the CDX environments used for customer demos.

**Comment by perikorese** [comment]
I am a freelance fullstack developer who wants to explore Power Apps in order to see if it's useful for a task, that I'm working on. I have found it to be extremely difficult to get a test/developer account for Power Apps/Power Platform. I am kind of moving in a loop through Microsoft help articles etc.
I created a free Azure account as I believed it would help, but no luck.

If you have any tips for me, I'd appreciate it.

**Reply by andrewconnell** [reply]
Unfortunately, the future of the dev program and those dev tenants is just not clear. Microsoft has dropped the ball on developers in not at least giving more information or an update. Maybe they've decided against offering the benefit... fine... just say that. But... **no answer or guidance after so long is disrespectful and to me, it communicates a lack of interest/care in the Microsoft 365 developer ecosystem.**

The best option I see forward at this time is the $6 Microsoft 365 Business Basic license as I mentioned [in this section](https://www.voitanos.io/blog/microsoft-365-developer-program-status-summer-2024/#going-forward---what-you-can-expect) of this article.

It's also a good option when, along with the metered Microsoft 365 Copilot Chat (*that I wrote about in my **[Microsoft 365 Copilot Chat Consumption Billing Developers](https://www.voitanos.io/blog/microsoft-365-copilot-chat-consumption-billing-developers/)** article*) if you want to do any custom agent development outside of the Copilot Studio option.

Personally, I don't think these paid options are too much to ask of a developer. It's the cost of doing business. I do wish that we had ways of loading sample content to make dev easier... ¯\_(ツ)_/¯

**Comment by andrewconnell** [comment]
Finally, I've got a good update for the Microsoft 365 developer tenants! 🎉

I brought your feedback to Microsoft's MVP Summit in March 2025 where I had the chance to raise this issue, or rather, make an impassioned plea. While I'm bound by NDA and can't share specific details of what I learned, here's what I can tell you:

I raised the issue & Microsoft's response was honest, direct, and clear, leaving me confident that they not only understand the issue but are actively working on it. While I don't think it's appropriate to disclose who at Microsoft answered the question, I can say that I trust the person who responded, they are in a position to impact change, and my follow-up conversations with them since the MVP Summit have reinforced this feeling.

**My takeaway**: I'm optimistic & anticipate we'll hear from Microsoft to learn more about the M365 developer program and dev tenants within the next few weeks/month.

**Comment by oliviercatteauIT** [comment]
Hi Andrew ,
Could i reproduce a sandbox with just 1  Microsoft 365 Business Premium license ?
I need to test different scenario, with some users ( 4 OR 5) but i don't want a big invoice at the end of the month, it's for learning. (All data could be deleted each day)
Thank you for this informations. It's help me a lot.
Olivier

**Reply by andrewconnell** [reply]
I touched on this in my updated article **[Microsoft 365 Developer Program Status - April 2025](https://www.voitanos.io/blog/microsoft-365-developer-program-status-april-2025/#current-options-for-developers)**

You can purchase a commercial license (or multiple licenses) on a month-to-month basis or an annual commitment. At a minimum, you can get a M365 Biz Essentials license for $7.20/month (with a month-to-month commitment) or $6.00/mo (with an annual commitment).
